src/elona/wish.cpp

```diff
@@ -344,12 +370,14 @@ bool grant_special_wishing(const std::string& wish)
     {
         txt(i18n::s.get("core.locale.wish.wish_man_inside"));
     }
-    else if (match_special_wish(wish, "ehekatl", {"ehekatl"}))
+    else if (auto pair = match_god_summon_wish(wish))
     {
-        txt(i18n::s.get("core.locale.wish.wish_ehekatl"));
+        auto god_id = pair->first.get();
+        auto chara_id = pair->second;
+        txt(i18n::s.get("core.locale.god." + god_id + ".summon"));
```

Comment:
`core.locale.god.core.{god ID without mod name}.summon` is missing. Also, this message is not required. Use `i18n::s.get_m_optional` instead.